third_party/xla_client/pjrt_computation_client.cc

```diff
@@ -79,13 +81,26 @@ std::vector<ComputationClient::DataPtr> PjRtComputationClient::TransferToServer(
   std::vector<ComputationClient::DataPtr> datas;
   datas.reserve(tensors.size());
   for (auto& tensor : tensors) {
-    xla::Literal literal(tensor.shape);
-    tensor.populate_fn(tensor, literal.untyped_data(), literal.size_bytes());
-
     PjRtDevice* pjrt_device = StringToPjRtDevice(tensor.device);
-    std::shared_ptr<xla::PjRtBuffer> buffer =
-        client_->BufferFromHostLiteral(literal, pjrt_device).ValueOrDie();
-    buffer->GetReadyFuture().Await();
+
+    auto literal = std::make_shared<xla::Literal>(tensor.shape);
+    tensor.populate_fn(tensor, literal->untyped_data(), literal->size_bytes());
+    absl::InlinedVector<int64_t, 4> byte_strides(
+        literal->shape().dimensions_size());
+    ShapeUtil::ByteStrides(literal->shape(), absl::MakeSpan(byte_strides));
```

Comment:
side question: can you confirm this works as expected for dynamic shape? I assume it should.

Reply:
How can I confirm this works with dynamic shape?

Reply:
Since none of this code has been tested with dynamic shapes, can you file an issue to me internally to look into it?

Reply:
Done. I sent you a ticket. Thanks.